src/main/java/frc/robot/commands/Vision/GoToAprilTagCommand.java

```diff
@@ -42,6 +43,8 @@ public void execute() {
             double distanceY = botPose[1];
             double rotationNeeded = botPose[5];
             // added -1 to stop 1 meter away from the apriltag
+            driveSubsystem.setSetpointArrivalX(distanceX);
```

Comment:
put it initialize, and you need to pass to it 1 meter, and at the calculate just give the current position